### PR TITLE
vulns(lodash & hoek)

### PR DESCRIPTION
**Pull request by vdeturckheim**

### CODE REVIEW

File: vuln/npm/367.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "id": 367,
+  "created_at": "2018-02-12",
+  "updated_at": "2018-02-12",
+  "title": "hoek prototype pollution",
+  "author": "Olivier Arteau (HoLyVieR)",
+  "module_name": "hoek",
+  "publish_date": "2018-02-12",
+  "cves": [
+    "CVE-2018-3721"
```
Comment (lirantal) [inline vuln/npm/367.json]: should be CVE-2018-3728

File: vuln/npm/367.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  ],
+  "vulnerable_versions": "<5.0.3",
+  "patched_versions": ">=5.0.3",
+  "slug": "lodash-prototype-pollution",
```
Comment (lirantal) [inline vuln/npm/367.json]: correct slug to hoek

File: vuln/npm/367.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  "overview": "hoek node module before 5.0.3 suffers from a prototype pollution vulnerability via 'merge' and 'applyToDefaults' functions, which allows a malicious user to modify the prototype of 'Object' via __proto__, causing the addition or modification of an existing property that will exist on all objects.",
+  "recommendation": "Update module to 5.0.3 or higher",
+  "references": "https://hackerone.com/reports/310439",
+  "cvss_vector": "CVSS:3.0/AV:L/AC:H/PR:H/UI:R/S:U/C:N/I:N/A:L",
+  "cvss_score": 1.8,
```
Comment (lirantal) [inline vuln/npm/367.json]: 3.8

File: vuln/npm/368.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "CVE-2018-3721"
+  ],
+  "vulnerable_versions": "<4.17.5",
+  "patched_versions": ">=4.17.5",
+  "slug": "lodash-prototype-pollution",
+  "overview": "lodash node module before 4.17.5 suffers from a prototype pollution vulnerability via 'defaultsDeep', 'merge', and 'mergeWith' functions, which allows a malicious user to modify the prototype of 'Object' via __proto__, causing the addition or modification of an existing property that will exist on all objects.",
+  "recommendation": "Update module to 4.17.5 or higher",
+  "references": "https://hackerone.com/reports/310443",
+  "cvss_vector": "CVSS:3.0/AV:L/AC:H/PR:H/UI:R/S:U/C:N/I:N/A:L",
+  "cvss_score": 1.8,
```
Comment (lirantal) [inline vuln/npm/368.json]: 2.5

File: vuln/npm/367.json (hunk 1 above)
Comment (vdeturckheim) [reply vuln/npm/367.json]: Good catch

File: vuln/npm/367.json (hunk 2 above)
Comment (vdeturckheim) [reply vuln/npm/367.json]: Good catch

File: vuln/npm/367.json (hunk 3 above)
Comment (vdeturckheim) [reply vuln/npm/367.json]: Reduced to 2.5 on H1. I dont' get why I can't have 1.8 with the same params as on other issue.

Comment (lirantal) [reply vuln/npm/367.json]: I e-mailed them an issue about this CVSS weirdness stuff, hope they're able to fix it sooner than later.

**Pull request merged**